### ISSUE TITLE
:GoRun does not respect default GOPATH (go 1.8+)

### ISSUE THREAD

**Issue by kron4eg** [fatih/vim-go#1208]
### Behavior

When I'm trying to `:GoRun` main that contains import of library that is in default `$GOPATH`, it doesn't work.

```
main.go:7:2: cannot find package "github.com/kezhuw/toml" in any of:
	/usr/local/go/src/github.com/kezhuw/toml (from $GOROOT)
	/Users/kron/src/github.com/kezhuw/toml (from $GOPATH)
```

Note that `github.com/kezhuw/toml` is in `~/go/src/github.com/kezhuw/toml/`.
### Steps to reproduce:

Using Go 1.8 run

    unset $GOPATH
    go get github.com/kezhuw/toml

using this Go file

```
package main

import (
	"fmt"
	"time"

	"github.com/kezhuw/toml"
)

func main() {
	data := []byte(`key = 2016-01-07T15:30:30.123456789Z`)
	var out struct{ Key time.Time }

	err := toml.Unmarshal(data, &out)
	if err != nil {
		panic(err)
	}

	fmt.Println(out.Key.Format(time.RFC3339Nano))
}
```

Run `:GoRun` and observe errors (vim-go seems to setup default GOPATH to $HOME).

Run
```
$ go run main.go
2016-01-07T15:30:30.123456789Z
```

### Configuration

* vim version: 8.0.0329
* vim-go version: latest master
* go version: go version go1.8 darwin/amd64

**Comment by fatih** [comment]
Hi. I've opened a PR with the fix: https://github.com/fatih/vim-go/pull/1248 can you please test it? Thanks

**Comment by fatih** [comment]
Fixed with #1248
